src/quickstart/provider/minikube/dns.md

```diff
@@ -31,6 +33,34 @@ $ host something-random.192.168.99.100.nip.io
 something-random.192.168.99.100.nip.io has address 192.168.99.100
 ```
 
+### Using DNSMasq
+
+If you `nip.io` is working for you, you can skip this section, and proceed to verify the hostname.
+
+If you prefer not to use `nip.io` or cannot (because your DNS provider might have blocked it), you can use `dnsmasq` on Linux and macOS or `Acrylic` on Windows.
```

Comment:
`dnsmasq` didn't work out-of-the-box for me on macOS until I manually added 127.0.0.1 to my DNS servers list--I gather this is a common issue when getting DNS via DHCP. Maybe it's worth adding a sentence here like:
```
You may need to ensure that the `dnsmasq` service at 127.0.0.1 is listed as a DNS server for your network connection.
```